tests/conftest.py

```diff
@@ -55,6 +55,7 @@ def app_config(app_config):
         "APP_ALLOWED_HOSTS": "localhost",
         "CELERY_TASK_ALWAYS_EAGER": True,
         "CDS_ILS_LITERATURE_UPDATE_COVERS": False,
+        "EXTEND_LOANS_LOCATION_UPDATED": False,
```

Comment:
@FlorianCassayre we were missing this in the tests, otherwise the `Inspect` was failing. Let's try to think to a better solution tomorrow if possible! :)

Reply:
Oh right, my bad 😕